CallSystemLinkDataServices/Readme.md

```diff
@@ -1,236 +1,228 @@
+
+   3. On the **Administrators** tab, ensure **Add the local Windows Administrators group to the admins role** is selected.
+   4. On the **HTTP Port** tab, set the **Server HTTP Port**.
+      > **Note**: This example assumes port 9090 is chosen.
+
+   5. On the **CORS** tab, ensure **Enable CORS for web servers running on the same machine as clients** is selected.
+   6. On the **Summary** tab, review your selections and then click **Finish**.
+
+### Building and Running the Example Web application
+
+1. Clone the [ni/webvi-examples](https://github.com/ni/webvi-examples) repository to your machine. [Go here](https://help.github.com/articles/cloning-a-repository/) if you’re new to cloning repositories on GitHub.
+2. Open the **Call SystemLink Data Services** example in LabVIEW NXG Web Module.
+   >**Note**: You can either search for the example by name in the search bar or select **Learning»Examples»Programming WebVIs»Call SystemLink Data Services** to launch it.
+
+3. Open `index.gviweb` and click **Run**.
+4. On the **Projects Files** tab, double-click the `Full Data Services App.gcomp` to open it.
+5. On the **Document tab**, click **Build** to build your web application.
+   > **Note**: You can automatically launch and view the web application locally by right-clicking the web application in SystemDesigner and then clicking **Run**.
+
+### Hosting Overview
+
+After you create a web application and build the package in LabVIEW NXG Web Module, you need to move it to a web server. This enables administrators to access it from a web browser. Copy your entire web application output directory to any web server you choose.
+> **Note:** To navigate to your web application output on your machine, click **Locate directory in Windows Explorer** on the **Document** tab of your web application component document. You can also access the build output by navigating to your web application’s project folder manually (`\CallSystemLinkDataServices\Builds`).
+
+Furthermore, this project includes a distribution document (`Full Data Services App.lvdist`), which you can use to build a package (`.nipkg`). A package is also required for hosting a web application on SystemLink Cloud. [Go here](http://www.ni.com/documentation/en/labview/3.0/application-builder/distributing-app-lib/) to learn more about distributing applications.
+
+### Local Hosting
+
+Follow the instructions below to host the web app on a web server.
+
+### Hosting a Package File (.nipkg) on the NI Web Server
+
+1. Open `CallSystemLinkDataServices.lvproject`
+2. Open `Full Data Services App.lvdist`.
+3. Click the **Build distribution**.
+   > **Note**: The button looks like a hammer.
+
+4. Click **Locate item in Windows Explorer** to find the build output.
+5. Double-click the package (`.nipkg`) and follow the on-screen instructions.
+6. Open a web browser and navigate to `http://localhost:9090/CallSystemLinkdataservices/index.html`.

```

Comment:
shouldn't need the `index.html` in the url

Reply:
Removed